routes/index.js

```diff
@@ -1,81 +1,138 @@
-const notes = require('../notes.js');
-const express = require('express');
+const notes = require("../src/notes.js");
+const express = require("express");
+const { schema, delSchema } = require("../validation/validation-schema.js");
+const {
+    validateAddModSchema,
+    validateDelSchema,
+} = require("../validation/validation.js");
 const router = express.Router();
 
 // Welcome Page
-router.get('/', (req, res) => {
-    res.send('<h1>Welcome to Notes App!</h1>');
+router.get("/", async (req, res) => {
+    res.send("<h1>Welcome to Notes App!</h1>");
 });
 
 // Listing the Notes
-router.get('/list', (req, res) => {
-    notes.loadNotes().then((data) =>{
-        res.status(200).send(data);
-    }).catch((err) =>{
-        res.status(400).send(err);
-        console.log(err);
-    })
+router.get("/list", async (req, res) => {
+    notes
+        .loadNotes()
+        .then((data) => {
+            res.status(200).send(data);
+        })
+        .catch((err) => {
+            res.status(500).send(err);
+            console.log(err);
+        });
```

Comment:
you can use async await with try/catch to do the same thing. Would be more easier to read.

Reply:
Fixed this. Kindly review!